cumulus_library/statistics/statistics_templates/count.sql.jinja

```diff
@@ -102,19 +101,19 @@ CREATE TABLE {{ table_name }} AS (
         {%- endif -%}
 
         {#- resource specific filtering conditions -#}
-        {%- if fhir_resource == 'documentreference' and filter_resource %}
+        {%- if fhir_resource == 'documentreference' %}
```

Comment:
Dropped the `filter_resource` check here because it's inside a giant `{% if filter_resource %}`